packages/gatsby-theme-docs/src/components/notifications.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import UnstyledInformationIcon from '../icons/information-icon.svg';
+import UnstyledWarningIcon from '../icons/warning.svg';
+import UnstyledErrorIcon from '../icons/error.svg';
+import createStyledIcon from '../utils/create-styled-icon';
+import markdown2React from '../utils/markdown-2-react';
+import { pxToRem, colors, tokens, dimensions } from '../design-system';
+
+const Container = styled.div`
+  display: flex;
+  width: 100%;
+  padding: ${dimensions.spacings.m};
+  margin: ${dimensions.spacings.s} 0;
+  border: ${pxToRem('1px')} solid ${colors.light.borderInfo};
+  border-radius: ${tokens.borderRadius6};
+`;
+
+const IconWrapper = styled.div`
+  flex-shrink: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: ${pxToRem('24px')};
+  height: ${pxToRem('24px')};
+`;
+
+const InfoTextWrapper = styled.div`
+  margin-left: ${dimensions.spacings.s};
+`;
+
+function Box(props) {
+  return (
+    <Container css={props.containerCustomStyle}>
+      <IconWrapper>{renderIcon(props.icon)}</IconWrapper>
+      <InfoTextWrapper>{markdown2React(props.children)}</InfoTextWrapper>
+    </Container>
+  );
+
+  function renderIcon(icon) {
+    const InformationIcon = createStyledIcon(UnstyledInformationIcon);
+    const WarningIcon = createStyledIcon(UnstyledWarningIcon);
+    const ErrorIcon = createStyledIcon(UnstyledErrorIcon);
+
+    switch (icon) {
+      case 'warning':
+        return <WarningIcon color="iconWarning" />;
+      case 'error':
+        return <ErrorIcon color="iconError" />;
+      default:
+        return <InformationIcon color="iconInfo" />;
+    }
+  }
+}
+
+Box.propTypes = {
+  containerCustomStyle: PropTypes.object.isRequired,
+  icon: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+};
+
+export function Info(props) {
+  const containerCustomStyle = css`
+    border-color: ${colors.light.iconInfo};
+    background-color: ${colors.light.surfaceInfo2};
+  `;
+
+  return (
+    <Box containerCustomStyle={containerCustomStyle} icon="default">
```

Comment:
```suggestion
    <Box css={containerCustomStyle} icon="default">
```

However, what we can also do is to pick the proper styles within the `Container` component, based on the given `icon` prop. This avoids to define the styles in every component.

```jsx
const Container = styled.div`
  border-color: ${props => {
    switch (props.icon) {
      case 'warning'
        return colors.light.iconWarning;
      case 'error'
        return colors.light.iconError;
      default
        return colors.light.iconInfo;
    }
  }};
  background-color: ${props => {
    switch (props.icon) {
      case 'warning'
        return colors.light.surfaceWarning;
      case 'error'
        return colors.light.surfaceError;
      default
        return colors.light.surfaceInfo2;
    }
  }};
`
```